docs/source/tutorials/instrument-modeling/equity-extension.rst

```diff
@@ -0,0 +1,186 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+How to Use the Equity Extension Package
+#######################################
+
+To follow the script used in this tutorial, you can
+`clone the Daml Finance repository <https://github.com/digital-asset/daml-finance>`_. In particular,
+the Equity test folder ``src/test/daml/Daml/Finance/Instrument/Equity/Test/`` is the starting point
+of this tutorial.
+
+How to Use the Equity Extension in Your Application
+***************************************************
+
+As explained in the :ref:`Getting Started <structure-of-code-dependencies>` section and on the
+:doc:`Architecture <../../overview/architecture>` page, your app should only depend on the interface
+layer of Daml Finance. For equities this means that you should only include the
+:doc:`equity interface package <../../packages/interfaces/daml-finance-interface-instrument-equity>`.
+
+Your initialization scripts are an exception, since they are only run once when your app is
+initialized. This creates the necessary factories. Your app can then create equity instruments
+through these factory interfaces.
+
+The Equity Interface
+********************
+
+The equity extension supports different lifecycle related events, for example dividends, stock
+splits and mergers. These are modeled using the choices on the
+:ref:`Equity interface <module-daml-finance-interface-instrument-equity-instrument-13224>`,
+namely ``DeclareDividend``, ``DeclareReplacement`` and ``DeclareStockSplit``. We will now
+demonstrate each one with a concrete lifecycle event.
+
+Dividend
+********
+
+The most common lifecycle event of an equity is probably dividends. This normally means that
+the holder of a stock receives a given amount of cash for each stock held. This is modeled using
+the ``DeclareDividend`` choice. It implements the
+:ref:`Distribution Event interface <module-daml-finance-interface-lifecycle-event-distribution-91943>`,
+which allows you to specify distribution of units of an instrument for each unit of a target
+instrument. In the case of a cash dividend, this would be a cash instrument. However, the
```

Comment:
This bit is slightly difficult to understand. It reads like the choice implements the distribution event interface. And the "distribution of units of an instrument for each unit of a target instrument" is a bit convoluted. Maybe just "distribution per share" is easier to understand.